dataactvalidator/config/sqlrules/a18_cross_file.sql

```diff
@@ -1,10 +1,26 @@
 SELECT
-    approp.row_number,
-    approp.tas,
-    approp.gross_outlay_amount_by_tas_cpe,
-    SUM(op.gross_outlay_amount_by_pro_cpe) as gross_outlay_amount_by_pro_cpe_sum
-FROM appropriation approp JOIN object_class_program_activity op
-ON approp.tas = op.tas AND approp.submission_id = op.submission_id
+	approp.row_number,
```

Comment:
Per @cmc333333's observation on A19, we also need to incorporate a change to A18 to handle some new math in the updated rule. I've Kristina for a clarification on this to make sure I'm grokking the rule text correctly.